asset-tracking-history-saver/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "asset-tracking-history-saver",
+  "version": "1.0.0",
+  "description": "CLI tool that saves enhanced location updates history from a channel to a file",
+  "main": "index.ts",
+  "scripts": {
+    "start": "node ./lib/index.js",
+    "build": "tsc -p ."
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "ably": "^1.2.4",
+    "commander": "^7.0.0",
+    "death": "^1.1.0",
+    "typescript": "^4.3.2"
```

Comment:
I think typescript should be a devDependency here? Or if you're putting build deps here then @types/node should be here too.